custom_components/wyzeapi/climate.py

```diff
@@ -142,11 +143,17 @@ def target_temperature_low(self) -> Optional[float]:
 
     @property
     def preset_mode(self) -> Optional[str]:
-        raise NotImplementedError
+        # pylint: disable=R1705
+        if self._thermostat.preset == Preset.HOME:
```

Comment:
Can we clean this up by doing a switch statement?

Reply:
Definitely can. I was just reverting back the changes that initially disabled presets in https://github.com/JoshuaMulliken/ha-wyzeapi/issues/207

Reply:
Ahh gotcha.